tests/helper/helper_oc.go

```diff
@@ -401,6 +401,7 @@ func (oc *OcRunner) GetEnvs(componentName string, appName string, projectName st
 		"-o", "jsonpath='{range .spec.template.spec.containers[0].env[*]}{.name}:{.value}{\"\\n\"}{end}'")
 
 	for _, line := range strings.Split(output, "\n") {
+		line = strings.TrimPrefix(line, "'")
```

Comment:
I noticed a `'` gets appended to the first line of the output of this command. Eg hello=world them the first line will be
`'hello=world`

Basically you can then read the map with 
```go
  val, ok := envs["'hello"]
```

This trims it out